jupyter_book/commands/create.py

```diff
@@ -0,0 +1,72 @@
+import sys
+import os.path as op
+
+import argparse
+
+from jupyter_book.create import new_book
+from jupyter_book.utils import str2bool
+
+
+def create():
+    parser = argparse.ArgumentParser(description="Create a new Jupyter Book")
+    parser.add_argument(
+        "name", help="The name of your Jupyter Book "
+                     "(your book template will be placed in a "
+                     "folder of this name)")
+    parser.add_argument("--out-folder", default='.',
+                        help="The location where your book will be placed")
+    parser.add_argument("--license", default=None,
+                        help="A path to a LICENSE.md file if you "
+                             "have already created one")
+    parser.add_argument("--content-folder", default=None,
+                        help="A path to a folder that holds your book content")
+    parser.add_argument("--toc", default=None,
+                        help="A path to a yaml file that contains a"
+                             " Table of Contents for your Jupyter Book."
+                             " This will overwrite parts of the book "
+                             "template's default toc.yml configuration")
+    msg = ("A path to a configuration YAML file that contains "
+           "configuration for your Jupyter Book. This will overwrite"
+           " parts of the book template's default _config.yml configuration")
+    parser.add_argument("--config", default=None,
+                        help=msg)
+    msg = ("A path to a CSS file that defines some custom CSS rules for"
+           " your book")
+    parser.add_argument("--custom-css", default=None, help=msg)
+    parser.add_argument("--custom-js", default=None,
+                        help="A path to a JS file that defines some "
+                             "custom CSS rules for your book")
+    parser.add_argument("--extra-files", default=None, nargs="+",
+                        help="A list of extra files / folders to copy"
+                             " into your book's directory")
+    parser.add_argument("--overwrite", default=False, action="store_true",
+                        help="Whether to overwrite a pre-existing book"
+                             " if it exists")
+    msg = ("Whether to build the book with demo content instead of your"
+           " own content")
+    parser.add_argument("--demo", default=False, action="store_true",
+                        help=msg)
+    parser.add_argument("--verbose", default='yes',
+                        help="Whether to display output information. [yes/no]")
+    args = parser.parse_args(sys.argv[2:])
+
+    ###############################################
+    # Default values and arguments
+
+    args = parser.parse_args(sys.argv[2:])
```

Comment:
Duplicate of L51 ?

Reply:
Instead of writing to `args` can we just set `dest`  for each of the options ?

Reply:
hmm ... does `dest` allow you to avoid `parser.parse_args`? I thought `dest` is inferred from the option string so you don't need to specify it

Reply:
Sorry, I shouldn't have used `dest`. I was thinking something like:
```
args = parser.parse_args(sys.argv[2:])
new_book(**vars(args))
```

Also, multiple variables are set from `toc` here, and it looks like we have `verbose` twice !

Reply:
That's really neat! But here, we have a couple of variables that shouldn't be passed as is. Perhaps something to iterate in future PRs?

Also, that was a good catch! The fact that the tests pass indicate that the tests should be improved.